lib/pymedphys/docs/lib/ref/index.rst

```diff
@@ -12,5 +12,4 @@ itself. Below is the documentation for a range of these libraries.
     dicom
     gamma
     metersetmap
-    electronfactors
```

Comment:
shouldn't the documentation be moved to experimental rather than deleted?
Alternatively, if you are truly deleting electronfactors, shouldn't it be removed from streamlit apps?
Maybe there is a different PR that did some of this?